.travis.yml

```
@@ -6,29 +6,45 @@ os:
   - windows
 julia:
   - 1.0  # LTS
-  - 1.3  # Latest release
+  - 1    # Latest release
```

Are we consistent with two spaces for comments, or to have them all line up?

```suggestion
  - 1  # Latest release
```

---

In this case I wanted them to line up

---

n this case I wanted them to line up